src/main/webapp/js/user-page-loader.js

```diff
@@ -42,6 +42,7 @@ function showMessageFormIfViewingSelf() {
             loginStatus.username == parameterUsername) {
           const messageForm = document.getElementById('message-form');
           messageForm.classList.remove('hidden');
+	  document.getElementById('about-me-form').classList.remove('hidden');
```

Comment:
style nit: to be consistent with the rest of the file, use spaces instead of tabs for indentation. Also, add another space to align this line.